requirements.txt

```diff
@@ -9,7 +9,7 @@ pytest~=7.1.2
 pytest-qt~=4.1.0
 
 # Development
-black~=22.1.0
+black~=22.3.0
```

Comment:
bug related to "click" in 22.1.0, fixed in 22.3.0 - see https://github.com/psf/black/issues/2964